osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceAttributes.cs

```diff
@@ -24,6 +24,9 @@ public class OsuPerformanceAttributes : PerformanceAttributes
         [JsonProperty("effective_miss_count")]
         public double EffectiveMissCount { get; set; }
 
+        [JsonProperty("deviation")]
```

Comment:
```suggestion
        [JsonProperty("total_deviation")]
```

just for consistency